apps/flame-defi/app/pool/modules/remove-liquidity/components/pool-fees-and-liquidity-card.tsx

```diff
@@ -8,7 +8,7 @@ export const PoolFeesAndLiquidityCard = ({
   className,
   liquidityToRemove,
 }: {
-  poolTokens: PoolToken[];
+  poolTokens: PoolToken[] | [];
```

Comment:
```suggestion
  poolTokens: PoolToken[];
```
`| []` is redundant, an empty array still satisfies `PoolToken[]`